CorsixTH/Lua/languages/english.lua

```diff
@@ -944,10 +949,6 @@ tooltip.calls_dispatcher = {
   close = "Close the calls dispatcher dialog",
 }
 
-tooltip.machine_window = {
-  toggle_machine_menu = "Click to open machine menu",
-}
-
```

Comment:
Verify no save state tries to rely on this (it probably really won't, unless an autosave somehow happens while hovering the tooltip it was for); if it does move it to the very bottom of file.